src/payment/strategies/digitalriver/digitalriver-payment-strategy.spec.ts

```diff
@@ -115,17 +115,28 @@ describe('DigitalRiverPaymentStrategy', () => {
     describe('#initialize()', () => {
         const digitalRiverLoadResponse = getDigitalRiverJSMock();
         const digitalRiverComponent = digitalRiverLoadResponse.createDropin(expect.any(Object));
+        const digitalRiverElement = digitalRiverLoadResponse.createElement(expect.any(Object), expect.any(Object));
         const customer = getCustomer();
         let options: PaymentInitializeOptions;
         let onErrorCallback: (error: OnCancelOrErrorResponse) => {};
         let onSuccessCallback: (data?: OnSuccessResponse) => {};
+        let container: HTMLDivElement;
 
         beforeEach(() => {
             jest.spyOn(store.getState().billingAddress, 'getBillingAddressOrThrow').mockReturnValue(getBillingAddress());
             jest.spyOn(store.getState().customer, 'getCustomer').mockReturnValue(customer);
             jest.spyOn(digitalRiverScriptLoader, 'load').mockReturnValue(Promise.resolve(digitalRiverLoadResponse));
             jest.spyOn(digitalRiverLoadResponse, 'createDropin').mockReturnValue(digitalRiverComponent);
+            jest.spyOn(digitalRiverLoadResponse, 'createElement').mockReturnValue(digitalRiverElement);
```

Comment:
Hi there, in `checkout-sdk-js`, it should be a blank line after `jest.spyOn()`([sample](https://github.com/bigcommerce/checkout-sdk-js/blob/49928fe40793871a7c99daa5bb1d29f798d760c5/src/payment/strategies/zip/zip-payment-strategy.spec.ts#L131)). Could we follow this style?

Reply:
Correct, I'll do it